### PR TITLE
feature/DT-1830-your-files-accessible-modal

### PR DESCRIPTION
**Pull request by chopkinsmade**
### Description of change
Switch the file uploader modals to use a `<dialog>` to be the same as the other modals used in the rest of data workspace

### Checklist

* [ ] Have tests been added to cover any changes?

### CODE REVIEW

File: dataworkspace/dataworkspace/static/js/react/features/your-files/popups/ErrorModal.js [hunk 1]
```diff
@@ -1,53 +1,63 @@
...
-                type="button"
-                onClick={onClose}
-                className="govuk-button govuk-button--secondary modal-button"
-              >
-                Close
-              </button>
+          <div className="modal-contents">
+            <div className="modal-body">
+              <div className="panel-body">
+                <p className="govuk-body" ng-if="error.errors.length">
```
Comment (ian-leggett) [inline dataworkspace/dataworkspace/static/js/react/features/your-files/popups/ErrorModal.js:21]: This condition is using an Angular directive. Can we change this to use React instead?

File: dataworkspace/dataworkspace/static/js/react/features/your-files/popups/ErrorModal.js [hunk 2]
```diff
@@ -1,53 +1,63 @@
...
+                  {error.message}
+                </p>
+                <table className="govuk-table" ng-if="error.errors.length">
```
Comment (ian-leggett) [inline dataworkspace/dataworkspace/static/js/react/features/your-files/popups/ErrorModal.js:24]: Same as above

**Review by ian-leggett — COMMENTED** [review]
Looks good. Just one thing, I noticed some conditions using Angular directives, can we remove this or do we need it?

File: dataworkspace/dataworkspace/static/js/react/features/your-files/popups/ErrorModal.js (hunk 1 above)
Comment (chopkinsmade) [reply dataworkspace/dataworkspace/static/js/react/features/your-files/popups/ErrorModal.js]: Doesn't seem to have any effect, removed and errors still appear

**Review by ian-leggett — APPROVED** [review]

**Pull request merged**